d2l-typography.js

```diff
@@ -1,7 +1,8 @@
+import '@brightspace-ui/core/components/colors/colors.js';
 import '@polymer/polymer/polymer-legacy.js';
-import 'd2l-colors/d2l-colors.js';
 import './d2l-typography-shared-styles.js';
-const importUrl = 'https://s.brightspace.com/lib/fonts/0.4.0/assets/';
+
+const importUrl = 'https://s.brightspace.com/lib/fonts/0.5.0/assets/';
 
 const $_fontFaces = document.createElement('style');
```

Comment:
Unfortunately I don't think we're able to completely replace the contents here yet since there are some components that pull this in (via `include="d2l-typography"`) to leverage some of the classes (ex. "d2l-heading-1") within their shadow-DOM.  In contrast, core's typography is applied to the `head` only, and constructible adopted/stylesheets are used.